pkg/commands/compute/validate.go

```diff
@@ -31,7 +31,7 @@ func (c *ValidateCommand) Exec(in io.Reader, out io.Writer) error {
 		return fmt.Errorf("error reading file path: %w", err)
 	}
 
-	if err := validate(p); err != nil {
+	if err := validate(p, nil); err != nil {
```

Comment:
I won't consider this a breaking change, but I'm personally not a fan of having to pass `nil`. This was why I had suggested a variadic function signature. 

But I appreciate this makes the code's intention clearer as far as we only ever expect there to be a _single_ validation step, so I can live with it.